app/assets/javascripts/gallery.js.erb

```diff
@@ -179,19 +179,19 @@ document.addEventListener("turbolinks:load", function(){
   /* ======= Home Page Functionality ======= */
   /* ======================================= */
   $(document).ready(function(){
-    var selector = '.tabs a'
+    var selector = '.page-home .tabs a'
     $(selector).on('click', function(){
-      $(selector).removeClass('active');
+      $(selector).closest("li.tab").removeClass('active');
```

Comment:
This is changing the home tabs to make the HTML classes consistent with bootstrap's way of dealing with tabs